bokeh/sphinxext/bokeh_plot.py

```diff
@@ -284,6 +283,12 @@ def _process_script(source, filename, env, js_name, use_relative_paths=False):
     return (script, js, js_path, source, c.doc)
 
 
+def _remove_module_docstring(source, doc):
+    # take docstring out of source so it doesn't show up twice
+    source = re.sub(rf'(\'\'\'|\"\"\")\s*{re.escape(doc)}\s*(\'\'\'|\"\"\")', "", source)
+    # ...and strip leading whitespace
+    return re.sub(r'^\s*', "", source)
```

Comment:
I think we can just use the built in `strip` for this: 
```
In [1]: "\n \n foo \n ".strip()
Out[1]: 'foo'
```
but also I would perfom this outside this helper (and even outside the `if doc:` conditional, we just jsut always strip leading or trailing whitespace, regardless)